functional-tests/envs/testenv.py

```diff
@@ -58,9 +56,29 @@ class StrataRunContext(flexitest.RunContext):
     """
 
     def __init__(self, datadir_root: str, name: str, env: flexitest.LiveEnv):
+        super().__init__(env)
         self.name = name
         self.datadir_root = datadir_root
-        super().__init__(env)
+        # get the bitcoind config
+        btc_svc = self.get_service("bitcoin")
+        bitcoind_config = BitcoindConfig(
+            rpc_url=f"http://localhost:{btc_svc.get_prop('rpc_port')}",
+            rpc_user=btc_svc.get_prop("rpc_user"),
+            rpc_password=btc_svc.get_prop("rpc_password"),
+        )
+        # get reth endpoint
+        reth_svc = self.get_service("reth")
+        reth_rpc_http_port = reth_svc.get_prop("eth_rpc_http_port")
+        rollup_cfg = env.rollup_cfg()
+
```

Comment:
Will have to rethink how this works when we support running the client without reth integrated, but this is fine for now.